src/util/fetch.js

```diff
@@ -1,15 +1,15 @@
 export const fetchJson = (url, options = {}) => {
-    const requestHeaders = new Headers({
+    const requestHeaders = options.headers || new Headers({
```

Comment:
Depends if it's `fetch` or a shim... We're in a gray area here. 

If `option.headers` is not a `Headers` instance, the user will already get an error that the `set` method isn't available.